index.js

```diff
@@ -684,7 +792,12 @@ module.exports = {
   mnemonicToRootKeypair,
   decodePaperWalletMnemonic,
   xpubToHdPassphrase,
+  deriveBootstrapAddress,
   packAddress,
+  deriveBaseAddress,
```

Comment:
I would rather have those newly added functions live under the "shelley" key and have them in a separate shelley.js file which would be imported into index.js index.js is getting already very long and hard to navigate.

Similarly - the byron counterparts of those newly added functions (packAddress, unpackAddress) can be kept in byron.js and moved up to the "byron" key in this exported object.

Reply:
I don't see value in having byron-specific commands (and naming these shelley-), as byron seems to be completely discontinued in a very short time.
Also if we want to split, I'd rather move out all address-related logic, not just shelley-secific.

Reply:
I see - indeed the deriveBootstrapAddress does the same as the old packAddress call - in that case ok, let's not split the namespaces but I would also get rid of the legacy packAddress function completely as whoever depends on it can migrate to deriveBootstrapAddress